typing/errortrace.ml

```diff
@@ -1,3 +1,18 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*  Xavier Leroy and Jerome Vouillon, projet Cristal, INRIA Rocquencourt  *)
+(*                                                                        *)
+(*   Copyright 1996 Institut National de Recherche en Informatique et     *)
```

Comment:
Looking at `git blame`, it seems that I wrote the corresponding code for the `Unification_trace` (in 2018). I would rather kept the copyright header not wildly inaccurate (the 1996 date in particular does seem strange). Could you also squash all check-typo commits into one?

Reply:
This makes sense – I've made these changes, so let me know if you're satisfied that the new copyright headers are to your satisfaction!

Reply:
That's look good (if the format is accepted by check-typo). You also need to rebase due to the change in toplevel/genprintval (sorry about that); then we can merge before another conflicting change sneak in.